SimCalorimetry/EcalEBTrigPrimProducers/plugins/EcalEBTrigPrimPhase2ESProducer.cc

```diff
@@ -0,0 +1,694 @@
+            } else if (i == 3) {
+              std::ostringstream oss;
+              oss << " 0x" << std::hex << data;
+              std::string result5 = oss.str();
+
+              st6.append(result5);
+              std::cout << "" << st6 << std::endl;
+            }
+          }
+        }
+      }
+
+      mapStrip_[1][id] = param;
+    }
+
+    if (dataCard == "TOWER_EE") {
+      gis >> std::dec >> id;
+
+      if (flagPrint_)
+        std::cout << dataCard << " " << std::dec << id << std::endl;
+
+      param.clear();
+      for (int i = 0; i < 2; i++) {
+        gis >> std::hex >> data;
+        param.push_back(data);
+
+        if (flagPrint_) {
+          if (i == 1) {
+            std::cout << "0x" << std::dec << data << std::endl;
+          } else {
+            std::cout << " " << std::dec << data << std::endl;
+          }
+        }
+      }
+
+      // std::cout<<std::endl ;
+      mapTower_[1][id] = param;
+    }
+
+    if (dataCard == "TOWER_EB") {
+      gis >> std::dec >> id;
+
+      if (flagPrint_)
+        std::cout << dataCard << " " << std::dec << id << std::endl;
+
+      param.clear();
+      for (int i = 0; i < 3; i++) {
+        gis >> std::dec >> data;
+
+        if (flagPrint_) {
+          std::cout << " " << std::dec << data << std::endl;
+        }
+
+        param.push_back(data);
+      }
+
+      // std::cout<<std::endl ;
+      mapTower_[0][id] = param;
+    }
+  }
+}
+
+std::vector<int> EcalEBTrigPrimPhase2ESProducer::getRange(
+    int subdet, int tccNb, int towerNbInTcc, int stripNbInTower, int xtalNbInStrip) {
+  std::vector<int> range;
+  if (subdet == 0) {
+    // Barrel
+    range.push_back(37);  // stccNbMin
+    range.push_back(73);  // tccNbMax
+    range.push_back(1);   // towerNbMin
+    range.push_back(69);  // towerNbMax
+    range.push_back(1);   // stripNbMin
+    range.push_back(6);   // stripNbMax
+    range.push_back(1);   // xtalNbMin
+    range.push_back(6);   // xtalNbMax
+  } else {
+    // Endcap eta >0
+    if (subdet > 0) {
+      range.push_back(73);   // tccNbMin
+      range.push_back(109);  // tccNbMax
+    } else {                 // endcap eta <0
+      range.push_back(1);    // tccNbMin
+      range.push_back(37);   // tccNbMax
+    }
+    range.push_back(1);   // towerNbMin
+    range.push_back(29);  // towerNbMax
+    range.push_back(1);   // stripNbMin
+    range.push_back(6);   // stripNbMax
+    range.push_back(1);   // xtalNbMin
+    range.push_back(6);   // xtalNbMax
+  }
+
+  if (tccNb > 0) {
+    range[0] = tccNb;
+    range[1] = tccNb + 1;
+  }
+  if (towerNbInTcc > 0) {
+    range[2] = towerNbInTcc;
+    range[3] = towerNbInTcc + 1;
+  }
+  if (stripNbInTower > 0) {
+    range[4] = stripNbInTower;
+    range[5] = stripNbInTower + 1;
+  }
+  if (xtalNbInStrip > 0) {
+    range[6] = xtalNbInStrip;
+    range[7] = xtalNbInStrip + 1;
+  }
+
+  return range;
+}
+
+// bool EcalEBTrigPrimPhase2ESProducer::getNextString(gzFile &gzf){
```

Comment:
Remove this commented out code or explain why it should stay in the comment.

Reply:
Removed